.circleci/config.yml

```diff
@@ -230,6 +230,7 @@ jobs:
     description: "Build Android Flutter integration tests"
     docker:
       - image: ghcr.io/cirruslabs/flutter:stable
+    resource_class: large
```

Comment:
Not sure what changed since only a patch version of flutter was released but android integration tests were consistently failing due to the daemon process getting killed. This + increasing the memory for that seems to have fixed the issue.